packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -109,37 +127,46 @@ export default function VisualEditor( { styles } ) {
 					{ __( 'Back' ) }
 				</Button>
 			) }
-			<div
+			<motion.div
 				ref={ mergedRefs }
 				className="editor-styles-wrapper"
-				style={ resizedCanvasStyles || desktopCanvasStyles }
+				animate={ animatedStyles }
+				initial={ desktopCanvasStyles }
 			>
-				<WritingFlow>
-					{ ! isTemplateMode && (
-						<div className="edit-post-visual-editor__post-title-wrapper">
-							<PostTitle />
-						</div>
-					) }
-					<BlockList
-						__experimentalLayout={
-							themeSupportsLayout
-								? {
-										type: 'default',
-										// Find a way to inject this in the support flag code (hooks).
-										alignments: themeSupportsLayout
-											? alignments
-											: undefined,
-								  }
-								: undefined
-						}
-					/>
-				</WritingFlow>
-			</div>
+				<AnimatePresence>
+					<motion.div
+						key={ isTemplateMode ? 'template' : 'post' }
+						initial={ { opacity: 0 } }
+						animate={ { opacity: 1 } }
+					>
```

Comment:
Why the need for a second div? Can't this be part of the `motion.div` above?

Reply:
It needs to be unmounted/remounted to trigger the exit transition and it needs to fade/in out just the content and not the frame.

Reply:
What's the difference visually? Fading out the frame or content would look the same?

Reply:
The frame resizes itself at the same time, if we remount it, we kill that animation.